### PR TITLE
CIP-0009 | Adjust preamble and structure w.r.t CIP-0001

### LINKED ISSUE #649: CIP-0009 | Adjust preamble and structure w.r.t CIP-0001
Fixed by https://github.com/cardano-foundation/CIPs/pull/650.

### PR DESCRIPTION
**Pull request by rphair**
Fixes #649.

([edited version](https://github.com/rphair/CIPs/blob/remediate-0009/CIP-0009/README.md))

### CODE REVIEW

**Comment by rphair** [comment]
@kevinhammond N.B. the copyright notice in the header was Creative Commons, while the document footer said Apache.  I'm guessing both that the Apache choice was made deliberately and that this was because of the code sections in the document, and that the CC in the header was probably copied indeliberately.  FYI I've set both to Apache.

File: CIP-0009/README.md [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
 CIP: 9
 Title: Protocol Parameters (Shelley Era)
-Authors: Kevin Hammond <kevin.hammond@iohk.io>
 Status: Active
-Type: Informational
+Category: Ledger
+Authors:
+  - Kevin Hammond <kevin.hammond@iohk.io>
+Implementors:
+  - IOG
```
Comment (Ryun1) [inline CIP-0009/README.md:9]:
```suggestion
  - IOG <https://iog.io/>
```

**Review by Ryun1 — APPROVED** [review]
LGTM

**Comment by kevinhammond** [comment]
> @kevinhammond N.B. the copyright notice in the header was Creative Commons, while the document footer said Apache. I'm guessing both that the Apache choice was made deliberately and that this was because of the code sections in the document, and that the CC in the header was probably copied indeliberately. FYI I've set both to Apache.

Yes Apache is consistent with other IOHK material and should be sufficiently permissive

**Review by Crypto2099 — APPROVED** [review]
LGTM

**Pull request merged**